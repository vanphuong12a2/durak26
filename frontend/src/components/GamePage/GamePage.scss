.game-page {
  max-width: 1000px;
  margin: auto;
  padding-top: 5px;
}
