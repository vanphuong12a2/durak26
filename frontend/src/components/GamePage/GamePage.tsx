import React from 'react';

interface Props {
  match: {params: {gameId: string}}
}

const GamePage = (props: Props) => {
  return (
    <div>Game page: {props.match.params.gameId}</div>
  );
};

export default GamePage;
