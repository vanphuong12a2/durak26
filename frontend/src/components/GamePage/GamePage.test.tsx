import React from 'react';
import {create} from 'react-test-renderer';
import GamePage from './GamePage';


describe('<GamePage />', () => {

  describe('Snapshots', () => {
    it('renders correctly', () => {
      const match = {params: {gameId: 'someId'}};
      const app = create(<GamePage match={match}/>);
      expect(app.toJSON()).toMatchSnapshot();
    });
  });

});
