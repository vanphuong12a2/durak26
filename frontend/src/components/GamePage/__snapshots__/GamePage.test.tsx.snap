// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GamePage /> Snapshots renders correctly 1`] = `
<div>
  Game page: 
  someId
</div>
`;
